src/components/WalletCard/WalletCard.js

```diff
@@ -113,30 +138,27 @@ class WalletCard extends Component<Props, StateProps> {
   }
 
   render() {
-    const {
-      data,
-      balance,
-      activeWalletId,
-    }: Props = this.props
-
     const {
       id,
       xpub,
-      customType,
+      type,
+      addressName,
+      derivationIndex,
       isSimplified,
-    }: Wallet = data
+      activeWalletId,
+    }: Props = this.props
 
     const {
       newName,
+      ethBalance,
       isNewNameUniq,
       isRenameActive,
     }: StateProps = this.state
 
-    const addressesCount: number = 5
     const isMultiAddress: boolean = !!xpub
-    const addressName: ?string = 'Address Name'
     const isActive: boolean = (id === activeWalletId)
     const hasMessage: boolean = (!isNewNameUniq && isRenameActive)
+    const name: string = addressName ? `${addressName}  •  ` : ''
```

Comment:
We have `src/utils/getAddressName` for get address name for multi address wallets